apps/web/src/pages/profile/penalties.tsx

```diff
@@ -1,9 +1,75 @@
 import MainLayout from "@/components/layout/MainLayout"
 import ProfileLayout from "@/components/layout/ProfileLayout"
 import { NextPageWithLayout } from "../_app"
+import { Accordion, AccordionContent, AccordionItem, AccordionTrigger } from "@dotkomonline/ui"
+import { DotFilledIcon } from "@radix-ui/react-icons"
+import PenaltyRules from "./penaltyRules"
+
+/* TODO - Set up connection to Users marks router
+ *  Remove all dummy marks
+ */
+
+const dummyMark = {
+  id: "",
+  title: "Første Mark",
+  givenAt: new Date(),
+  updatedAt: new Date(),
+  category: "",
+  details: "manglende møte på Dåp",
+  duration: 2000000,
+}
+const dummyMark1 = {
+  id: "",
+  title: "Andre Mark",
+  givenAt: new Date(),
+  updatedAt: new Date(),
+  category: "",
+  details: "flexing av hook med 05",
+  duration: 9000000000009,
+}
+const dummyMark2 = {
+  id: "",
+  title: "Tredje Mark",
+  givenAt: new Date(),
+  updatedAt: new Date(),
+  category: "Sosialt",
+  details: "manglende prikker",
+  duration: 100000,
+}
+
+const listOfDummyMarks = [dummyMark, dummyMark1, dummyMark2]
+const emptylist = []
 
 const PenaltiesPage: NextPageWithLayout = () => {
-  return <div>Penalties</div>
+  return (
+    <div className="ml-3 flex flex-col space-y-12">
+      <p className="text-[28px] font-medium">Prikker</p>
+      <div className="flex flex-col">
+        <p className="text-[24px] font-medium">Aktive Prikker</p>
+        {activePenalties}
+      </div>
+      <div>
+        <p className="text-[24px] font-medium">Gamle Prikker</p>
+        {oldPenalties}
+      </div>
+      <div className="flex flex-col space-y-2">
+        <p className="text-[24px] font-medium">Suspensjoner</p>
+        {suspensions}
+      </div>
+      <div>
+        <p className="text-[24px] font-medium">PrikkeRegler</p>
+        <Accordion className="max-w-[590px]" type="single" collapsible>
```

Comment:
same as above

Reply:
this could probably be moved to it's own component

Reply:
Changed to text-2xl instead